package.json

```diff
@@ -1,80 +1,53 @@
 {
-  "name": "react-scroll-sync",
-  "version": "0.0.0-development",
-  "description": "Synchronize scroll positions across multiple scrollable containers",
-  "main": "dist/index.js",
-  "scripts": {
-    "clean": "rimraf ./dist",
-    "prebuild": "npm run lint",
-    "build": "NODE_ENV=production npm run clean && webpack",
-    "prepare": "npm run build",
-    "lint": "eslint src --ext .js,.jsx",
-    "lint:fix": "npm run lint -- --fix",
-    "lint:staged": "lint-staged",
-    "styleguide:server": "styleguidist server",
-    "styleguide:build": "styleguidist build"
-  },
-  "lint-staged": {
-    "*.js": [
-      "eslint --fix",
-      "git add"
-    ]
-  },
-  "pre-commit": "lint:staged",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/okonet/react-scroll-sync.git"
-  },
-  "keywords": [],
-  "author": "Andrey Okonetchnikov <andrey@okonet.ru>",
-  "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/okonet/react-scroll-sync/issues"
-  },
-  "homepage": "https://github.com/okonet/react-scroll-sync#readme",
-  "eslintConfig": {
-    "extends": "okonet",
-    "rules": {
-      "react/forbid-prop-types": 0,
-      "react/jsx-wrap-multilines": 0
+    "name": "react-scroll-sync",
+    "version": "0.11.2",
```

Comment:
How is the release process look like now? I'd like to switch to changesets if possible.

Reply:
My thinking in changing it 0.11.2 was so that `npm link` could work when I was editing. However, I can revert back to 0.0.0-development for now. I am not familiar with changesets but am open to collaborating and learning more

Reply:
Yes let's revert for now since it's how semantic release works. We can switch to changesets as a follow up